package.json

```diff
@@ -47,6 +47,7 @@
     "nyc": "15.0.0",
     "prettier": "^1.19.1",
     "source-map-support": "0.5.16",
+    "ts-loader": "^6.2.1",
```

Comment:
Why did we add `ts-loader` as a dependency?

Reply:
This should of been included on the next PR step, removing from here.